values-schema.yaml

```diff
@@ -2366,27 +2367,56 @@ properties:
     additionalProperties: false
     properties:
       zones:
-        description: Extra dns zones that the cluster can administer (see dns). Team services can use this to publish their URLs on.
+        description: Extra dns zones that the cluster can administer. Team services can use this to publish their URLs on.
         items:
           $ref: '#/definitions/domain'
         type: array
+      domainFilters:
```

Comment:
Looks like property rename. If so please add it to the values-changes.yaml file.

Reply:
of course